README.md

```diff
@@ -127,13 +127,13 @@ conflict in ports will be resolved automatically.
 Start bootstrap node by creating a network
 
 ```shell
-$ ./nknd -test create
+$ ./nknd -c
 ```
 
 Start other nodes by joining the network
 
 ```shell
-$ ./nknd -test join
+$ ./nknd -seed $RemoteNodeIP:$HttpJsonPort
```

Comment:
Is there any commend for normal user joining the network without set any parameter?

Reply:
@dreamfly281 SeedList is a MUST parameter which the entry points during you join a ring from. It can not be omitted. Previous cmdline "-test join" just implied SeedList setting via config.json.
New cmdline "--seed" just provide another approach to override SeedList in config.json. If --seed indicate an empty string "", the setting in config.json is still efficient.
```
if len(seedStr) > 0 {
    // Just override config.json when --seed length > 0
    config.Parameters.SeedList = strings.Split(seedStr, ",")
}
```
Currently, the default value of --seed is "localhost:30003"

Reply:
We can have another line like:
To join the network using the seed list defined in config.json
./nknd

Reply:
@dreamfly281 @yilunzhang I supplemented a fix 107457a for used config.json setting as default when no --seed params provided.